[CS2113T-W12-2] UniMods

Poopies99 opened the pull request:
NUS Mods Lite offers an exciting new way to access module information and timetable from a local offline save, greatly reducing frustrations borne out of RAM-Eating Chrome and deleted cache.

ChewyChair committed:
Implement interrupt for update command.

ChewyChair committed:
Add search interrupt.

xuanlc113 committed:
add print for timetable info

xuanlc113 committed:
add print for timetable info

ChewyChair committed:
Implement a better interrupt.

Poopies99 committed:
Introduced Event Command and corresponding methods and exceptions

ChewyChair committed:
Add javaDocs.

Poopies99 committed:
Remove Magic Numbers in Command Parser and Other small changes to comply with java coding standards

Poopies99 committed:
Made changes to deleteFromSchedule and deleteFromLessons method in timetable to reduce nested function calling

adricpjw committed:
v1 UI for printing useritems

aditichadha1310 committed:
added Unofficial Transcript and fixed CAP calculation

added Unofficial Transcript and fixed CAP calculation

aditichadha1310 committed:
Updated User Guide

Added transcript section to the UG

xuanlc113 committed:
fix merge conflict with upstream

xuanlc113 committed:
fix merge conflict with upstream

xuanlc113 committed:
fix commandparser conflict

adricpjw committed:
Merge pull request #96 from xuanlc113/change-semester

Add command to change semester

ChewyChair committed:
Merge pull request #90 from ChewyChair/master

Add ability to stop running search/update commands.

Poopies99 committed:
Merge pull request #4 from AY2122S1-CS2113T-W12-2/master

Update

Poopies99 committed:
Merge branch 'master' of https://github.com/Poopies99/tp into Add_Feature

 Conflicts:
	src/main/java/seedu/parser/CommandParser.java

ChewyChair committed:
Refactor duke to unimods and add logo.

ChewyChair committed:
Fixed text-ui-test and build.gradle.

xuanlc113 committed:
add exam dates in timetable

xuanlc113 committed:
add timetable show user items only

xuanlc113 committed:
add remove timetablelesson test

xuanlc113 committed:
modify timetablecommand test

aditichadha1310 committed:
solved checkstyle errors

solved checkstyle errors

adricpjw committed:
Merge pull request #97 from ChewyChair/master

Refactor duke to unimods and add logo.

adricpjw committed:
Merge pull request #98 from xuanlc113/exam-info

add exam dates in timetable

adricpjw committed:
Merge branch 'master' into view-personal-tasks

adricpjw committed:
Merge pull request #99 from xuanlc113/view-personal-tasks

View personal tasks

aditichadha1310 committed:
Refactored the code using string formatting

ChewyChair committed:
Re-enable assertions.

ChewyChair committed:
Merge pull request #100 from ChewyChair/master

Re-enable assertions.

adricpjw committed:
Merge branch 'master' into Branch-Transcript

adricpjw committed:
Merge pull request #95 from aditichadha1310/Branch-Transcript

added Unofficial Transcript and fixed CAP calculation

adricpjw committed:
fixed some prior merge errors and refactored printSchedule

adricpjw committed:
Merge pull request #102 from adricpjw/fix/desync-merge-errors

fixed some prior merge errors and refactored printSchedule

ChewyChair committed:
Bugfix for double prompt cursor when cancelling search/update.

ChewyChair committed:
Merge pull request #103 from ChewyChair/master

Bugfix for double prompt cursor when cancelling search/update.

Poopies99 committed:
Merge pull request #5 from AY2122S1-CS2113T-W12-2/master

Update Code

Poopies99 committed:
Changes to event

Poopies99 committed:
Merge branch 'Add_Feature' of https://github.com/Poopies99/tp into Add_Feature

Poopies99 committed:
Added Event Option in AddCommand

Poopies99 committed:
Finished Add Command and Edit Command

Poopies99 committed:
Resolve Checkstyle errors

ChewyChair committed:
Merge pull request #93 from Poopies99/Add_Feature

Added Event Feature to allow user to add personal events and task

adricpjw committed:
added visualisation to personal tasks, added easy deletion of personal tasks

aditichadha1310 committed:
Merge pull request #94 from adricpjw/dev/TimetableUI-UserItems

Dev/timetable UI user items

adricpjw committed:
made help command more robust by creating properties for each

adricpjw committed:
fixed checkstyle

adricpjw committed:
added quickfix for non-lectures

ChewyChair committed:
Add examdate when showing modules.

ChewyChair committed:
Merge branch 'enh/HelpCommand-Robust' of https://github.com/adricpjw/tp

adricpjw committed:
fixed add bug again

sean-phang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+- [Appendix: Instructions for manual testing]()
+    * [Launch and shutdown]()
+    * [Deleting a person]()
+    * [Saving data]()
```
Might be better to not include links that are not in the DG

DuckWillow replied:
I agree, this code should be in .puml file.

sean-phang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+
+![LogicComponent](./uml-diagrams/LogicComponent.png)
+
+The **Logic** component. </br>
```
The '\</br>' probably does not seem to work as a line break

sean-phang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+`AddCommand`,  `ClearCommand`, `DeleteCommand`, `ExitCommand`, `HelpCommand`, `SearchCommand`, 
+`ShowCommand`, `TimetableCommand` and the `UpdateCommand` classes.
+
+![LogicComponent](./uml-diagrams/LogicComponent.png)
```
The diagram probably should not be placing the labels in the box in the top-right corner. The other components should probably also not have the box as it is probably the wrong notation. I am also unsure about the weird icons on the red, 'command' boxes

![image](https://user-images.githubusercontent.com/19592521/139175545-e7005d00-217b-45c0-a3ed-2c8a19dc27fe.png)

zh1huang replied:
Agree with this, could remove the icons and label to make it cleaner

sean-phang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+
+![](resources/ModulesClassDiagram.png)
+
+####Parsing and Saving of Weeks via Gson
```
Space between # and the text is missing

DuckWillow replied:
![image](https://user-images.githubusercontent.com/71874189/139193647-63ce8e61-73e2-4883-bf54-077b116638aa.png)
This is not well-formatted.

sean-phang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+An example of a lesson in the *timetable* array in the special term, where *weeks* contains an object with its own 
+*start* and *end* keys.
+
+By default Gson does not handle keys that have multiple data types. If `Weeks` is defined as an object, Gson will throw 
+an error when parsing normal semester lessons as it expects an array, and similarly, if weeks is defined as an array 
+in `Lessons` then Gson will throw an error when parsing Special term lessons as it expects an object instead.
```
special terms such as 'weeks' and 'lessons' are sometimes not stylized, this might cause confusion on what is an object.

sean-phang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+By utilizing a TypeAdapter for Weeks in Lessons, a  custom deserialization method for Gson to use for this specific key 
+was implemented.
+
+![](uml-diagrams/WeeksRead.png)
```
Should WeeksAdapterFactory create() WeeksAdapter line be solid instead?

![image](https://user-images.githubusercontent.com/19592521/139176624-d0b55e02-68a8-4c99-89c2-9aaa0813c6a0.png)

sean-phang commented on docs/DeveloperGuide.md:
````diff
@@ -1,38 +1,266 @@
+"size": 25,
+"covidZone": "Unknown"
+}
+```
````
Good examples given

DuckWillow commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+The Architecture Diagram given below explains the high-level design of the App. 
+Given below is a quick overview of main components and how they interact with each other.
+![Unimods](./resources/architecture.png)
+
```
![image](https://user-images.githubusercontent.com/71874189/139192963-95d06f61-1b13-4e38-8cd4-6b6e3f50bbf3.png)
Is this correct? Should it be a dotted arrow from storage point to timetable instead?

DuckWillow commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+emulates that of the json, with each `Module` containing an `Attributes` object and an array of `Semester` objects, 
+`Semester` containing an array of `Lesson` objects, and  `Lesson` containing an array of `Weeks` objects.
+
+![](resources/ModulesClassDiagram.png)
```
![image](https://user-images.githubusercontent.com/71874189/139193454-61e2ca73-ac52-47bc-a4c3-5fab0bf4d931.png)
A class diagram should include attribute type.

yzhedwin replied:
![image](https://user-images.githubusercontent.com/69493564/139370009-eaacab6a-4323-4d31-b7cf-d31012893a0c.png)
Should this be '0..1' instead?

yzhedwin replied:
Would be better to state the type and accessibility of each attributes in each class?

yzhedwin replied:
Is there a need to place weeks on the composition line?

DuckWillow commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+
+#### Search
+
+![](uml-diagrams/Search.png)
```
![image](https://user-images.githubusercontent.com/71874189/139194081-9dff0538-370d-4697-9304-f22e26d11a66.png)
The diagram is too complicated.

DuckWillow reviewed (commented):
Week 11 DG review.

ChewyChair committed:
Bugfix for AddCommand.

ChewyChair committed:
Merge branch 'master' of https://github.com/ChewyChair/tp

ChewyChair committed:
Fix errors in sequence diagrams.

ChewyChair committed:
Fix a checkstyle error.

ChewyChair committed:
Update text-ui-test

zh1huang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+
+**Commons**
+> **Commons** represents a collection of classes used by multiple other components.
+The rest of the App consists of five components:
```
![image](https://user-images.githubusercontent.com/59475240/139370053-bc5c385b-63b3-4307-ab68-669e4082165b.png)

Consider splitting another section for "The rest of the App consists of five components::"

zh1huang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+    * [Deleting a person]()
+    * [Saving data]()
+
 ## Acknowledgements
```
Might have to add GSON under acknowledgements, together with AB-2/3

zh1huang commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
 
-## Product scope
-### Target user profile
+![](uml-diagrams/Update.png)
```
![image](https://user-images.githubusercontent.com/59475240/139370802-d385d6af-e7c5-4e61-9ac6-daa643582c81.png)

Block for [No connection] should not be empty?

yzhedwin commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
 
-{Describe the value proposition: what problem does it solve?}
+#### Initialization
+![](uml-diagrams/TimetableLoad.png)
```
![image](https://user-images.githubusercontent.com/69493564/139371212-52428bba-3169-41f6-9f6c-d8a375dc0f1c.png)

Should there be ":" prefix for the classes in this diagram?

yzhedwin replied:
Is the entity boxes below needed?

yzhedwin replied:
Should the life line end after object deletion?

zh1huang reviewed (commented):
Generally a decent job done!

yzhedwin commented on docs/DeveloperGuide.md:
```diff
@@ -1,38 +1,266 @@
+`Timetable` is converted to a `TimetableDto` object in order to separate different `TimetableItem` types. The `TimetableDto` object is then saved to a local json file via Gson.
 
-## Glossary
+#### Add
```
![image](https://user-images.githubusercontent.com/69493564/139372038-5484db77-b4cc-45b4-a656-7f1af7460b6d.png)
Would be better to be clearer on defining attributes. Perhaps can highlight the attributes. For example, is lesson here an attribute? Is Lesson a class? Might cause ambiguity due to the format

yzhedwin replied:
Should addModuleList() and addLesson() be termed methods or functions?

Poopies99 committed:
1. Resolve bugs mentioned by my lovely teammates.

2. Resolve bug in clear command that didn;t actually clear the timetable

Poopies99 committed:
Resolve checkstyle

Poopies99 committed:
Merge pull request #106 from Poopies99/Add_Feature

resolve bugs

ChewyChair committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp

# Conflicts:
#	src/main/java/seedu/command/AddCommand.java

ChewyChair committed:
Merge pull request #105 from ChewyChair/master

Bugfixes

Poopies99 committed:
Change question sequence

ChewyChair committed:
Add some changes to DG.

ChewyChair committed:
Bugfixes and improvements for v2.1.

ChewyChair committed:
Some UG fixes.

ChewyChair committed:
Abstract some functions to make them clearer.

ChewyChair committed:
Add error message for tampering.

ChewyChair committed:
Fix checkstyle violation.

ChewyChair committed:
Update text-ui-test to conform to Github Actions runner timezone.

ChewyChair committed:
Minor changes to text-ui-test.

ChewyChair committed:
Merge pull request #177 from ChewyChair/branch-v2.1

v2.1 bugfixes

ChewyChair committed:
Update config.yml and UserGuide.

ChewyChair committed:
Merge pull request #178 from ChewyChair/branch-v2.1

Update config.yml and UserGuide.

ChewyChair committed:
Resolve error in config file.

ChewyChair committed:
Testing theme.

ChewyChair committed:
Testing theme.

ChewyChair committed:
Merge branch 'branch-v2.1' of https://github.com/ChewyChair/tp into branch-v2.1

ChewyChair committed:
Merge pull request #179 from ChewyChair/branch-v2.1

(Hopefully) Resolve emojis not showing on ghPages

ChewyChair committed:
Revert change to theme.

ChewyChair committed:
Merge pull request #180 from ChewyChair/branch-v2.1

Revert change to theme.

ChewyChair committed:
Set theme jekyll-theme-cayman

ChewyChair committed:
Update to UG.

xuanlc113 committed:
fix change semester non-numerical input

xuanlc113 committed:
add javadocs

xuanlc113 committed:
add timetable storage load message

adricpjw committed:
fixing merges

adricpjw committed:
fixed a bunch of timetable-related issues such as events not clearing/not being saved to storage etc

adricpjw committed:
fixed checkstyle errors

Poopies99 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into Add_Feature

adricpjw committed:
Merge pull request #181 from xuanlc113/change-semester

fix change semester non-numerical input

adricpjw committed:
Merge pull request #182 from xuanlc113/add-javadocs

Add javadocs

adricpjw committed:
Fixed io redirection test

adricpjw committed:
fixed invalid input during add module resulting in incorrect UI behavior

adricpjw committed:
Merge branch 'master' into fix/Timetable-Issues

adricpjw committed:
fixed checkstyle error in javadocs

adricpjw committed:
Merge branch 'fix/Timetable-Issues' of github.com:adricpjw/tp into fix/Timetable-Issues

adricpjw committed:
fixed io redirection test

adricpjw committed:
add input for io-direction test for no profile loaded

Poopies99 committed:
Improve Code Quality and Readability from @EthanWong's comments

ChewyChair committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into branch-v2.1

adricpjw committed:
fixed io redirection test

ChewyChair committed:
Merge pull request #183 from adricpjw/fix/Timetable-Issues

LGTM!

ChewyChair committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into branch-v2.1

ChewyChair committed:
Refactor FileErrorException into its own class, add update to help command, and add/update javaDocs.

Poopies99 committed:
Remove Add Command Bugs, made changes to UserGuide to reflect how add command works

Poopies99 committed:
Added Documentation and checkstyle error

ChewyChair committed:
Update to error handling for module jsons.

Poopies99 committed:
Added addition test cases for Add Command

Poopies99 committed:
Fix Gradle Error

Poopies99 committed:
Fix Gradle Error

Poopies99 committed:
1. Added more loggers into Add Command and AddUI
2. General Code Cleanup

Poopies99 committed:
Code Clean up Timetable.java

Poopies99 committed:
Code Clean up

Poopies99 committed:
Fix a bug where Sectional Teaching lessons are displayed twice instead of only once when adding module

Poopies99 committed:
Updated UserGuide.md to reflect changes in output

Poopies99 committed:
1. Fix bug where modules not offered this semester seems to be added into timetable
2. Fix Formatting for Lesson Display

Poopies99 committed:
Fix Checkstyle errors

aditichadha1310 committed:
Fixed some bugs-1

ChewyChair committed:
Start on PPP.

Poopies99 committed:
Merge branch 'master' into Add_Feature

Poopies99 committed:
Merge pull request #184 from Poopies99/Add_Feature

Fix bugs, change UserGuide and added documentation

ChewyChair committed:
Add appendix to DG.

ChewyChair committed:
Updated UG.

ChewyChair committed:
Updates to documentation.

ChewyChair committed:
Add diagrams and updates to DG.

aditichadha1310 committed:
removed code for storage as it was already implemented on the upstream

Poopies99 committed:
Merge pull request #186 from ChewyChair/branch-v2.1

Updates to documentation

aditichadha1310 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into Bug-Fixes-1

aditichadha1310 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into Bug-Fixes-1

Poopies99 committed:
Updated DG and UG

aditichadha1310 committed:
clear modules from timetable and transcript

Implemented flags feature in clear command

aditichadha1310 committed:
Included personal particulars in the transcript

aditichadha1310 committed:
No duplicate modules can be stored in transcript

Poopies99 committed:
Added AddCommand Sequence Diagram for DG

ChewyChair committed:
Rename PPP.

aditichadha1310 committed:
Fixed the delete command

Poopies99 committed:
Added AddCommand.png

aditichadha1310 committed:
Fixed check-style error

Poopies99 committed:
Added AddCommand.png

Poopies99 committed:
Updated DG for addCommand

Poopies99 committed:
Added Poopies.md

aditichadha1310 committed:
added UG

aditichadha1310 committed:
removed magic strings

adricpjw committed:
updated documentation for UGDG and PPP

ChewyChair committed:
Merge pull request #188 from Poopies99/Add_Feature

Code Clean up + DG & UG update

aditichadha1310 committed:
Merge branch 'master' into Bug-Fixes-1

ChewyChair committed:
Merge branch 'master' into docs/DG-PPP-update

ChewyChair committed:
Merge pull request #189 from adricpjw/docs/DG-PPP-update

updated documentation for UGDG and PPP

aditichadha1310 committed:
Merge branch 'master' into Bug-Fixes-1

Poopies99 committed:
Refactor Poopies.md to poopies99.md

aditichadha1310 committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into Bug-Fixes-1

Poopies99 committed:
Fix Bug when adding modules

Poopies99 committed:
Fix Wrong Output Bug for Add Command

Poopies99 committed:
Fix Wrong Time Format Bug when adding Event

Poopies99 committed:
Fix Wrong Time Format Bug when adding Event

ChewyChair committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into branch-v2.1

Poopies99 committed:
1. Fix bug which doesn't verify each lesson with the same class number whether it conflicts with the timetable
2. Fix bug which doesn't show the correct event time when you add an event after 2300

ChewyChair committed:
Updates to DG.

ChewyChair committed:
Update to UG for flag clarity.

Poopies99 committed:
1. Fix bug which causes events have occur through the entire day to not be displayed
2. Updated the UG

Poopies99 committed:
Fix Checkstyle errors

Poopies99 committed:
Merge pull request #190 from ChewyChair/branch-v2.1

Updates to UG and DG

ChewyChair committed:
Changes to PPP.

ChewyChair committed:
Corrected Module class diagram.

ChewyChair committed:
Add v2.1 issues to PPP.

ChewyChair committed:
Merge pull request #191 from Poopies99/Add_Feature

Final Farewell PR from Poopies

ChewyChair committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into branch-v2.1

aditichadha1310 committed:
Updated UG for clear,delete,remove, transcript commands

aditichadha1310 committed:
Merge branch 'Bug-Fixes-1' of https://github.com/aditichadha1310/tp into Bug-Fixes-1

adricpjw committed:
updated docs readme to point to our PPP

adricpjw committed:
Merge pull request #193 from adricpjw/docs/Readme

updated docs readme to point to our PPP

aditichadha1310 committed:
Implemented grades functionality

aditichadha1310 committed:
Added PPP

aditichadha1310 committed:
Merge branch 'master' into Bug-Fixes-1

xuanlc113 committed:
add ug timetable flag, semester command

xuanlc113 committed:
update timetable user item image

xuanlc113 committed:
update timetable user item image

ChewyChair committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into branch-v2.1

ChewyChair committed:
Clean up links.

ChewyChair committed:
Merge pull request #192 from ChewyChair/branch-v2.1

Updated Module class diagram in DG.

aditichadha1310 committed:
Merge branch 'master' into Bug-Fixes-1

ChewyChair committed:
Merge pull request #185 from aditichadha1310/Bug-Fixes-1

Fixed some bugs-1

ChewyChair committed:
Fix table in AboutUs.md.

ChewyChair committed:
Merge pull request #194 from ChewyChair/branch-v2.1

Fix table in AboutUs.md.

ChewyChair committed:
Fixes to UG display bugs.

ChewyChair committed:
Trying to fix stubborn md syntax refusing to render on GH pages.

ChewyChair committed:
Attempt #2 to fix bug

ChewyChair committed:
Attempt #3 at fixing bug

Poopies99 committed:
Update poopies99.md

Poopies99 committed:
Update poopies99.md

ChewyChair committed:
Another sad attempt

Poopies99 committed:
Update poopies99.md

Poopies99 committed:
Update poopies99.md

Poopies99 committed:
Update poopies99.md

Poopies99 committed:
Update poopies99.md

Poopies99 committed:
Update poopies99.md

ChewyChair committed:
Try try again

xuanlc113 committed:
add dg timetable component

ChewyChair committed:
Again

xuanlc113 committed:
fix ug conflict

ChewyChair committed:
IM LOSING MY MIND

Poopies99 committed:
Update UserGuide.md

xuanlc113 committed:
add xuanlc113 ppp

ChewyChair committed:
Merge pull request #195 from xuanlc113/docs

Docs

ChewyChair committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into branch-v2.1

# Conflicts:
#	docs/UserGuide.md

Poopies99 committed:
Update poopies99.md

ChewyChair committed:
UG fixes

ChewyChair committed:
Standardize shells.

ChewyChair committed:
Add a brief description of each section.

ChewyChair committed:
Changes to aboutus and team readme.

ChewyChair committed:
Merge pull request #196 from ChewyChair/branch-v2.1

Standardize shell formats, add everyone to AboutUs and team README

ChewyChair committed:
Remove unused sections from DG

ChewyChair committed:
Merge pull request #197 from ChewyChair/branch-v2.1

Remove unused sections from DG

aditichadha1310 committed:
added remove command

aditichadha1310 committed:
Updated DG

aditichadha1310 committed:
Merge branch 'master' into branch-docs

aditichadha1310 committed:
Merge pull request #198 from aditichadha1310/branch-docs

Branch docs

aditichadha1310 committed:
finalTouchUp

aditichadha1310 committed:
commit-2

aditichadha1310 committed:
Merge pull request #199 from aditichadha1310/Branch-finaltouchup

finalTouchUp

aditichadha1310 committed:
added page break

aditichadha1310 committed:
Merge pull request #200 from aditichadha1310/branch-PPP

added page break

ChewyChair committed:
Add page breaks to DG, minor changes.

ChewyChair committed:
Add some more page breaks to DG.

ChewyChair committed:
Merge pull request #201 from ChewyChair/branch-v2.1

Add page breaks to DG.

aditichadha1310 committed:
added changes to PPP

aditichadha1310 committed:
Merge pull request #202 from aditichadha1310/branch-Final

added changes to PPP

ChewyChair committed:
Add missing hyperlink to release page in UG.

ChewyChair committed:
Merge pull request #203 from ChewyChair/branch-v2.1

Add missing hyperlink to release page in UG.

ChewyChair committed:
Standardize to UniMods, remove some extra unnecessary line.

ChewyChair committed:
Merge pull request #204 from ChewyChair/branch-v2.1

Standardize to UniMods, remove some extra unnecessary line.

ChewyChair committed:
Add Gson to acknowledgements, json -> JSON.

ChewyChair committed:
Merge pull request #205 from ChewyChair/branch-v2.1

Add Gson to acknowledgements, json -> JSON.

ChewyChair committed:
Some more fixes to DG.

ChewyChair committed:
Merge pull request #206 from ChewyChair/branch-v2.1

Some more fixes to DG.

Poopies99 committed:
Update UserGuide.md

ChewyChair committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into branch-v2.1

# Conflicts:
#	docs/UserGuide.md

ChewyChair committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-W12-2/tp into branch-v2.1

# Conflicts:
#	docs/UserGuide.md

ChewyChair committed:
Merge pull request #207 from ChewyChair/branch-v2.1

CS2101 Updates

aditichadha1310 committed:
Create UserGuide.md

ChewyChair committed:
Update UserGuide.md

ChewyChair committed:
Revert "Create UserGuide.md"

This reverts commit b514f027fec095d81fa2162933375e963eae41a8.

ChewyChair committed:
Readded changes.

ChewyChair committed:
Merge branch 'branch-v2.1' of https://github.com/ChewyChair/tp into branch-v2.1

ChewyChair committed:
Merge pull request #208 from ChewyChair/branch-v2.1

CS2101 UG changes

Poopies99 committed:
Update UserGuide.md

Poopies99 committed:
Update UserGuide.md
